src/CGALCache.h

```diff
@@ -8,7 +8,7 @@
 class CGALCache
 {
 public:
-	CGALCache(size_t limit = 100*1024*1024);
+	CGALCache(size_t limit = 100 *1024 *1024);
```

Comment:
It's up to you, but do you think it would make more sense if there were spaces on both sides of the operators?

Reply:
I saw that, I think this is a bug in uncrustify where it mistakes this to be a pointer as it's inside a function declaration. Couldn't find my way around it.

Reply:
Ahh ok. After I commented on this line, I scrolled down further and saw that in most places it had spaces on both sides as expected.